### PR TITLE
{Release} Upgrade to Azure CLI 2.64.0

### PR DESCRIPTION
**Pull request by azclibot**
Updated Azure CLI version.

### CODE REVIEW

**Comment by yonzhan** [comment]
Release

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,87 @@
 Release History
 ===============
 
+2.64.0
+++++++
+
+**ACR**
+
+* Improve deprecation message for `az acr helm` (#29700)
```
Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az acr helm`: Improve deprecation message (#29700)
```

Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az acr helm`: Improve deprecation message  (#29700)
```

Comment (calvinhzy) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* `az acr helm`: Improve deprecation message (#29700)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,87 @@
...
+* `az hdinsight azure-monitor-agent disable`: Disable the Azure Monitor Agent logs integration on an HDInsight cluster (#29752)
+
+**MySQL**
+
+* `az mysql flexible-server maintenance`: New command group for managing maintenance of MySQL flexible server (#29505)
+* `az mysql flexible-server update`: Remove storage passing for server update (#29739)
+
+**NetAppFiles**
+
+* Update max for usageThreshold (#29624)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:62]:
```suggestion
* `az netappfiles volume create/update`: Update max for usageThreshold (#29624)
```

Comment (kairu-ms) [inline src/azure-cli/HISTORY.rst:62]:
```suggestion
* `az netappfiles volume create/update`: Update max for `--usage-threshold` (#29624)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,87 @@
...
+* `az containerapp create/up`: Make `--target-port` optional (#29702)
+* `az containerapp env create`: Deprecate unused argument `--docker-bridge-cidr` (#29746)
+* `az containerapp job stop`: Deprecate option to stop list of given job executions (#29728)
+* Fix #29711: `az containerapp logs show`: Fix JSON escaping by default or with `--format json` (#29767)
+
+**HDInsight**
+
+* `az hdinsight create`: Support setting IP tags when creating HDInsight cluster (#29752)
+* `az hdinsight update`: Support update the managed identity of the cluster (#29752)
+* `az hdinsight azure-monitor-agent show`: Get update the managed identity of the cluster (#29752)
```
Comment (evelyn-ys) [inline src/azure-cli/HISTORY.rst:51]:
```suggestion
* `az hdinsight update`: Support updating the managed identity of the cluster (#29752)
* `az hdinsight azure-monitor-agent show`: Support getting the managed identity of the cluster (#29752)
```

File: src/azure-cli/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,87 @@
...
+
+**AKS**
+
+* `az aks nodepool update`: Add `--enable/disable-fips-image` flags for GA mutable fips (#29695)
+* `az aks create/update`: Support UserAssigned Managed Identity for grafana linking in managed prometheus (#29713)
+* `az aks create/update`: Update the VM SKU regex validation to include larger set of VMs for Azure Container Storage (#29726)
+
+**App Service**
+
+* `az staticwebapp create` and `az staticwebapp update`: Add `Dedicated` as supported SKU (#29514)
```
Comment (calvinhzy) [inline src/azure-cli/HISTORY.rst:21]:
```suggestion
* `az staticwebapp create/update`: Add `Dedicated` as supported SKU (#29514)
```

Comment (yanzhudd) [inline src/azure-cli/HISTORY.rst:21]:
```suggestion
* `az staticwebapp create/update`: Add `Dedicated` as supported SKU (#29514)
```

File: src/azure-cli/HISTORY.rst (hunk 2 above)
Comment (calvinhzy) [inline src/azure-cli/HISTORY.rst:62]:
```suggestion
* `az netappfiles volume create/update`: Update max quota for `--usage-threshold` (#29624)
```

**Review by AllyW — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by bebound — APPROVED** [review]

**Pull request merged**